app/ide-desktop/lib/client/src/project-management.ts

```diff
@@ -50,6 +50,7 @@ export function importProjectFromPath(openedPath: string): string {
         // Otherwise, we need to install it first.
         if (rootPath == null) {
             const message = `File '${openedPath}' does not belong to the ${common.PRODUCT_NAME} project.`
+            // eslint-disable-next-line no-restricted-syntax
```

Comment:
should be avoidable by wrapping the `return` below in an `else {}`. alternatively, if the lint rule isn't useful, it can be removed completely